askama/src/lib.rs

```diff
@@ -523,7 +523,7 @@ pub trait Template {
         if let Some(error) = adapter.done() {
             return Err(Error::Io(error));
         }
-        return outcome;
+        outcome
```

Comment:
Please change these in the originating commits instead of adding a separate commit. You can use `git rebase -x "cargo clippy --workspace --all-targets -- -D warnings"` to help with this.

Reply:
OK, given I have added quite a few commits, and I'll have to apply the change manually each time, I'll squash all commits into just two:  implementation + tests.